src/main/java/salesforce/ui/pages/classic/contracts/ClassicCreatedContractPage.java

```diff
@@ -0,0 +1,42 @@
+package salesforce.ui.pages.classic.contracts;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import salesforce.ui.pages.BasePage;
+import salesforce.ui.pages.classic.ClassicHomePageObjects;
+
+public class ClassicCreatedContractPage extends BasePage {
+
+    @FindBy(css = "div[id*=\"ctrc7_ileinner\"]")
+    private WebElement accountName;
```

Comment:
It is better to have the type of elements in the name for example accountNameTxtBox